waspc/data/Generator/templates/react-app/src/queries/_query.ts

```diff
@@ -0,0 +1,11 @@
+{{={= =}=}}
+import { createQuery } from './core'
+{=& operationTypeImportStmt =}
+
+
+const query = createQuery<{= operationTypeName =}>(
+  '{= queryRoute =}',
+  {=& entitiesArray =},
+)
+
+export default query
```

Comment:
Although we decided to abandon default exports (see #546 for details), we're still postponing it. It requires many  breaking changes in our API (and the docs).

In the meantime, I separated the export from the definition because it helps with VS Code's "on hover" feature.

When exporting the value directly:
![image](https://user-images.githubusercontent.com/22752307/230348944-853c6e47-3f9d-44b2-803d-46b18acdbab2.png)

When exporting the variable after the assignment:
![image](https://user-images.githubusercontent.com/22752307/230349204-d92f97a6-7528-4643-91fc-3f660f14e193.png)